Non-breaking space before sorting symbols

Augustin82 opened the pull request:
Replaces the regular space between a column name and the sorting symbol with a non-breaking space, preventing wonky wrapping when the column's name is wider than the column's content.

Augustin82 committed:
Non-breaking space before sorting symbols

gaul commented on src/Table.elm:
```diff
@@ -245,12 +245,12 @@ simpleTheadHelp (name, status, onClick) =
 darkGrey : String -> Html msg
 darkGrey symbol =
-  Html.span [ Attr.style [("color", "#555")] ] [ Html.text (" " ++ symbol) ]
+  Html.span [ Attr.style [("color", "#555")] ] [ Html.text (" " ++ symbol) ]
```
More clear to use HTML entity, e.g., `&nbsp;`?

billstclair committed:
elm-upgrade to Elm 0.19.

Fix shadowing and toString problems.

Change now nonexistent  functions to floating point math.

Add a CSS file for the example.

Update live sample links to billstclair.github.io.

billstclair committed:
Fix code link. Bump to 1.0.1.

billstclair committed:
It's Elm 0.19, not 0.18.

billstclair committed:
Bump version to 1.0.2.

billstclair committed:
Add a <tr> around the <th>s in the <thead> of the table.

This was reported in https://github.com/evancz/elm-sortable-table/issues/15

Bump package version from 1.0.2 to 1.0.3.
